packages/mongo-nav/src/org-nav/GovBanner.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import GovBuildingIcon from '@leafygreen-ui/icon/dist/GovBuilding';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Body } from '@leafygreen-ui/typography';
+
+const desktopContainer = css`
+  margin-bottom: 10px;
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  height: 5px;
+  background-color: #edf5fa;
+  align-items: flex-start;
+  justify-content: center;
+`;
+
+const tabletContainer = css`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  align-items: flex-start;
+  justify-content: center;
+  height: 0;
+`;
+
+const mobileContainer = css`
+  width: 30px;
+  height: 30px;
+  background-color: #edf5fa;
+  margin-left: 12px;
+  border-radius: 7px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
```

Comment:
These elements aren't clickable, so they shouldn't have this in any screen size.

Reply:
They're clickable in mobile/tablet

Reply:
Desktop shows the cursor, and mobile / tablet won't have the hover states, so we don't need this rule, right?

Reply:
I think we do need this rule, because otherwise the cursor doesn't indicate that the banner/tooltip are interactive

Reply:
With the logo (for example) we have `cursor:pointer`